packages/forms-web-app/src/controllers/examination/submitting-for/utils/get-back-link-url.js

```diff
@@ -0,0 +1,25 @@
+const { isQueryModeEdit } = require('../../../utils/is-query-mode-edit');
+const { getUserHasInterestedPartyNumber } = require('../../session/deadline/helpers');
+const { getExaminationApplicantValue } = require('../../session/deadline/details/applicant');
+
+const {
+	routesConfig: {
+		examination: {
+			directory,
+			pages: { applicant, checkYourAnswers, yourInterestedPartyNumber }
+		}
+	}
+} = require('../../../../routes/config');
+
+const getBackLinkUrl = (query, session) => {
+	if (isQueryModeEdit(query)) return `${directory}${checkYourAnswers.route}`;
```

Comment:
single return using the redirect URL like do everywhere else ?

I see the point in throwing the error. If this was a case we would need to do this everywhere else ?